test/spec/serverStorage.js

```diff
@@ -0,0 +1,49 @@
+var serverStorage  = require('../../lib/server/serverStorageUtil').storage;
+
+describe('serverStorage', function () {
+    describe('get', function () {
+        it('correctly returns value from storage if key is found',  function ()  {
+            serverStorage.set('testKey', 'testValue');
+            expect(serverStorage.get('testKey')).toBe('testValue');
+        });
```

Comment:
nit: This file has `4` spaces instead of the usual `2`. Lets be consistent.

Reply:
Ops!